### ISSUE TITLE
Installing previous version (1.10.2) returns 403 error

### ISSUE THREAD

**Issue by jyoonsong** [meteor/meteor#12034]
- The version of Meteor showing the problem: 1.10.2
- The operating system you're running Meteor on: M1 Mac OS

I want to install the previous version of the meteor, specifically version 1.10.2, on my mac.
However, it returns 403 error as follows.

```shell
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  7496    0  7496    0     0  78548      0 --:--:-- --:--:-- --:--:-- 85181
Downloading Meteor distribution
curl: (22) The requested URL returned error: 403                              

Retrying download in 5 seconds...
```
I tried `arch -x86_64 npm install -g meteor@1.10.2` but did not work either.
Are you not supporting the installation of Meteor 1.x anymore?
Thank you in advance

**Comment by StorytellerCZ** [comment]
For installing via NPM, the earliest package version available is `2.3.1`.
For the other installation version check out this: https://docs.meteor.com/expired-certificate.html

**Comment by StorytellerCZ** [comment]
Since there was no reply from the opener I have closed the issue.
